Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Starting elecciones as inkarri
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Starting elecciones as inkarri
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/elecciones/../bin/gunicorn", line 11, in <module>
    sys.exit(run())
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 185, in run
    super(Application, self).run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 71, in run
    Arbiter(self).run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 169, in run
    self.manage_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 477, in manage_workers
    self.spawn_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 542, in spawn_workers
    time.sleep(0.1 * random.random())
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 209, in handle_chld
    self.reap_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Starting elecciones as inkarri
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Starting elecciones as inkarri
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Exception in worker process:
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/workers/base.py", line 114, in init_process
    self.wsgi = self.app.wsgi()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/util.py", line 356, in import_app
    __import__(module)
  File "/home/inkarri/Proyectos/vpython3/elecciones/elecciones/wsgi.py", line 13
    from django.core.wsgi import get_wsgi_application}
                                                     ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/inkarri/Proyectos/vpython3/elecciones/../bin/gunicorn", line 11, in <module>
    sys.exit(run())
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 185, in run
    super(Application, self).run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/app/base.py", line 71, in run
    Arbiter(self).run()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 169, in run
    self.manage_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 477, in manage_workers
    self.spawn_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 542, in spawn_workers
    time.sleep(0.1 * random.random())
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 209, in handle_chld
    self.reap_workers()
  File "/home/inkarri/Proyectos/vpython3/lib/python3.4/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
WORKER TIMEOUT (pid:2033)
WORKER TIMEOUT (pid:2034)
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
WORKER TIMEOUT (pid:2078)
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
Starting elecciones as inkarri
